### PR TITLE
Remove debug alert

### PR DESCRIPTION
**Pull request by rarila**
Thanks @GDmac

### CODE REVIEW

File: app/view/lib/bolt/obj-stack.js [hunk 1]
```diff
@@ -17,7 +17,6 @@ var Stack = Backbone.Model.extend({
         // In the modal dialog, to navigate folders..
         $('#selectImageModal-stack').on('click', '.folder', function (e) {
             e.preventDefault();
-            alert('hoi');
```
Comment (bobdenotter) [inline app/view/lib/bolt/obj-stack.js]: Lol! :wave:

Comment (rarila) [inline app/view/lib/bolt/obj-stack.js]: And nobody noticed that for a long time. Is this as with TODOs. If that nobody notices for half a year the function can be removed? ;-)

Comment (GawainLynch) [inline app/view/lib/bolt/obj-stack.js]: So much #1555 here it's not funny

**Comment by GDmac** [comment]
What wonders me slightly more is, the alert never shows up, does it?
i couldn't find ``$('#selectImageModal-stack .modal-content')`` content in views anywhere? 
we only seem to have ``selectModal``.

**Pull request merged**